roles/vault/README.md

```diff
@@ -70,7 +73,7 @@ Example playbook (used with OpenStack Kayobe)
       consul_bind_ip: "{{ internal_net_ips[ansible_hostname] }}"
       consul_vip_address: "{{ internal_net_vip_address }}"
       vault_bind_address: "{{ external_net_ips[ansible_hostname] }}"
-      vault_vip_url: "{{ external_net_fqdn }}"
+      vault_vip_address: "{{ external_net_fqdn }}"
```

Comment:
I guess this guy is difficult to name - could be an ip address or a url, this is consistent though